client/src/pages/SitterDetailPage/SitterDetailPage.js

```diff
@@ -1,3 +1,382 @@
+// import React, { useState, useEffect } from 'react';
```

Comment:
Lol what happened to this whole file?

Reply:
Oh, I forget to tell there was some error that new created user's detail isn't loaded well after Tongil's commit.
I simply replaced with my local code for further works and it was committed accidentally.
As it has been fixed, I will pull them and not commit this one.

Reply:
Ah okay thanks!

Reply:
Yes, I fixed that. sorry for the inconvenience.